src/subscription_manager/repofile.py

```diff
@@ -615,7 +617,11 @@ def fix_content(self, content):
             zypper_cont['gpgcheck'] = '0'
 
         # See BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1858231
-        zypper_cont['repo_gpgcheck'] = '0'
+        if self.repo_gpgcheck:
```

Comment:
Please remove this line: `if self.repo_gpgcheck`

Reply:
Because its currently not supported in pulp to have signed repository metadata, I would not change the current behavior to always disable the repo_gpgcheck.

Reply:
I will redo this. But I created a default in the configuration file

Reply:
done